tfx/examples/airflow_workshop/setup/dags/taxi_utils.py

```diff
@@ -322,10 +322,9 @@
 #             for i in range(num_dnn_layers)
 #         ])
 #
-#   # TODO(b/158106209): This log path might change in the future.
-#   log_dir = os.path.join(os.path.dirname(fn_args.serving_model_dir), 'logs')
+#   # Write logs to path specfied by fn_args
 #   tensorboard_callback = tf.keras.callbacks.TensorBoard(
-#       log_dir=log_dir, update_freq='batch')
+#       log_dir=fn_args.log_dir, update_freq='batch')
```

Comment:
just noticed this will make the examples failed with release tfx libraray

we want the examples works with head and release tfx

so let's do 

TODO(b/158106209): deprecate xxx after next release	

if fn_args has log dir: [1]
  log_dir=fn_args.log_dir
else 
  log_dir = os.path.join(os.path.dirname(fn_args.serving_model_dir), 'logs')

[1]not sure if you can just use fn_args.log_dir when fn_args doesn't contains log_dir field in release tfx